src/components/content/systemStatus/SystemStatusBar.tsx

```diff
@@ -8,10 +8,10 @@ import React from 'react';
 import appStyles from '../../../styles/app.module.css';
 import { healthStatus } from '../../../xpanse-api/generated';
 import SystemStatusIcon from './SystemStatusIcon';
-import { useHealthCheckStatusQuery } from './useHealthCheckStatusQuery';
+import { useStackCheckStatusQuery } from './useStackStatusCheckQuery';
 
 function SystemStatusBar(): React.JSX.Element {
-    const healthCheckQuery = useHealthCheckStatusQuery();
+    const healthCheckQuery = useStackCheckStatusQuery();
```

Comment:
this must use the old useHealthCheckStatusQuery still. Just the new data model is used if necessary. API call must not be changed